.azuredevops/pipeline-integration-test.yml

```diff
@@ -0,0 +1,196 @@
+pr: none
+trigger: none
+
+pool:
+  vmImage: "ubuntu-latest"
+
+parameters:
+  - name: env
+    displayName: Environment to Run Test
+    type: string
+    values:
+      - dev
+      - sit
+    default: sit
+
+variables:
+  archiveName: nodo-dei-pagamenti-NRT.zip
+  destinationFolderExtractArtifact: nodo/extracted
+  deploy-pool: nodopagamenti-agent-pool-sit
+
+stages:
+  - stage: prepare_env
+    displayName: "Prepare env (${{ parameters.env }})"
+    variables:
+      - group: nodo-integration-test-${{ parameters.env }}
+    jobs:
+      - job:
+        displayName: "Prepare env (${{ parameters.env }})"
+        steps:
+          - checkout: self
+            clean: true
+            persistCredentials: true
+
+          - task: AzureKeyVault@2
+            inputs:
+              connectedServiceName: $(kv-service-connection-${{ parameters.env }})
+              keyVaultName: $(az-kv-name-${{ parameters.env }})
+              secretsFilter: "*"
+              runAsPreJob: true
+
+          - script: |
+              function replace {
+                  file=$1
+                  key=$2
+                  value=$3
+                  
+                  contents=$(jq "$key = \"$value\"" $file) && echo "${contents}" > $file
+              }
+
+              file=$(Build.SourcesDirectory)/src/integ-test/bdd-test/resources/config.json
+
+              # replace $file ".services.\"nodo-dei-pagamenti\".url"           "https://10.230.8.5:443/nodo-pagamenti/api/v1"
+              # replace $file ".services.\"nodo-dei-pagamenti\".healthcheck"   "/monitor/health"
+              # replace $file ".services.\"nodo-dei-pagamenti\".soap_service"  "/webservices/input"
+              # replace $file ".services.\"nodo-dei-pagamenti\".rest_service"  "/nodo-per-pm/v1"
+              # replace $file ".services.\"nodo-dei-pagamenti\".refresh_config_service"  "/config/refresh/CONFIG"
+
+              replace $file ".services.\"nodo-dei-pagamenti\".url"           "https://10.230.8.5:443/nodo"
```

Comment:
use a [private DNS](https://pagopa.atlassian.net/wiki/spaces/PAG/pages/459636892/Configurazione+API+Nodo+su+APIM+pagoPA#APIM-pagoPA-private-APIM-IPs) (ex. api.dev.platform.pagopa.it) instead of the IP static private APIM IP